components/kyma-environment-broker/internal/avs/client.go

```diff
@@ -186,6 +186,11 @@ func (c *Client) execute(request *http.Request, allowNotFound bool, allowResetTo
 			return c.execute(request, allowNotFound, false)
 		}
 		return response, fmt.Errorf("avs server returned %d status code twice for %s (response body: %s)", http.StatusUnauthorized, request.URL.String(), responseBody(response))
+	case http.StatusInternalServerError,
```

Comment:
I think all http errors with code grater or equal to 500 should be handled this way